src/scenes/levelend-scene/levelend-scene.ts

```diff
@@ -88,6 +67,7 @@ export class LevelEndScene {
   toggleLevelEndBackground = (shouldShow: boolean) => {
     if (this.levelEndElement) {
       this.levelEndElement.style.display = shouldShow ? 'block' : 'none';
+      this.levelEndElement.style.zIndex = "11";
```

Comment:
Please add comment what is the purpose of this zIndex

Reply:
+1

Reply:
added comments here. pls check